src/sentry/web/frontend/accounts.py

```diff
@@ -122,6 +122,14 @@ def recover_confirm(request, user_id, hash, mode="recover"):
                 if mode == "relocate":
                     user.username = form.cleaned_data["username"]
                     user.is_unclaimed = False
+                    # Relocation form required users to accept TOS and privacy policy
+                    # TODO(team-ospo/#228): Include imported org slugs here
+                    terms_accepted.send_robust(
```

Comment:
I thought we were going to iterate over each org the user is a part of?

Reply:
I think we decided on iterating over the imported org slugs here, but I looked into it and it seems to be a larger lift that I initially expected. I added a TODO there to tackle it later. Do you think this is blocking here?

Reply:
Take 5 min to try the following. If it does not work, let's leave the TODO.

```suggestion
                    # Only need first membership, since all of user's orgs will be in the same region.
                    membership = OrganizationMemberMapping.object.filter(user=user).first()
                    mapping = OrganizationMapping.object.get(id=membership.organization_id)
                    # Will need to modify the following method's signature to accept `str | None` for last arg).
                    orgs = org_service.get_organizations_by_user_and_scope(mapping.region_name, request.user)
                    for org in orgs:
                        terms_accepted.send_robust(
                            user=request.user
                            organization=org,
                            # ...
```

Reply:
Oh right, I forgot an assumption we make is that a relocation is done on an orgless user, so we can just find all their orgs they are a part of 🤦‍♂️. I'll do this

Reply:
thanks for writing this out